xml/System.Windows.Forms/AccessibleEvents.xml

```diff
@@ -1,1064 +1,1084 @@
+      <MemberSignature Language="C++ CLI" Value="SystemMenuEnd" />
+      <MemberType>Field</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Forms.AccessibleEvents</ReturnType>
+      </ReturnValue>
+      <MemberValue>5</MemberValue>
+      <Docs>
+        <summary>A menu from the menu bar was closed. The system sends this event for standard menus. Servers send this event for custom menus.</summary>
+      </Docs>
+    </Member>
+    <Member MemberName="SystemMenuPopupEnd">
+      <MemberSignature Language="C#" Value="SystemMenuPopupEnd" />
+      <MemberSignature Language="ILAsm" Value=".field public static literal valuetype System.Windows.Forms.AccessibleEvents SystemMenuPopupEnd = int32(7)" />
+      <MemberSignature Language="DocId" Value="F:System.Windows.Forms.AccessibleEvents.SystemMenuPopupEnd" />
+      <MemberSignature Language="VB.NET" Value="SystemMenuPopupEnd" />
+      <MemberSignature Language="F#" Value="SystemMenuPopupEnd = 7" Usage="System.Windows.Forms.AccessibleEvents.SystemMenuPopupEnd" />
+      <MemberSignature Language="C++ CLI" Value="SystemMenuPopupEnd" />
+      <MemberType>Field</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Forms.AccessibleEvents</ReturnType>
+      </ReturnValue>
+      <MemberValue>7</MemberValue>
+      <Docs>
+        <summary>
+          <para>A shortcut menu was closed. The system sends this event for standard menus. Servers send this event for custom menus. When a shortcut menu is closed, the client receives this message followed almost immediately by the <see langword="SystemMenuEnd" /> event. The system does not send the event consistently.</para>
+          <para>For a call to <c>TrackPopupMenu()</c>, a client will see <see langword="EVENT_SYSTEM_MENUSTART" /> followed almost immediately by <see langword="EVENT_SYSTEM_MENUPOPUPSTART" /> for the popup being shown.</para>
+        </summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+For a call to TrackPopupMenu(), a client will see `EVENT_SYSTEM_MENUSTART`
+             followed almost immediately by `EVENT_SYSTEM_MENUPOPUPSTART` for the popup being shown.
+
+          ]]></format>
+        </remarks>

```

Comment:
Enums cannot have remarks (only in triple slash).
I think this can be fixed in the tool directly. For now, I'm giving suggestions to remove them:

```suggestion
        </summary>
```